### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by suhail-codenia** [livewire/livewire#8713] [1. Help] [answered]
How do i install it in my laravel project ?

**Comment by nadeemkhan** [comment, marked as answer]
> How do i install it in my laravel project ?

You should type -> composer require livewire/livewire

**Comment by joshhanley** [comment]
@suhail-codenia I would recommend reading the docs, it has all the details. I would start here https://livewire.laravel.com/docs/quickstart

**Reply by suhail-codenia** [reply]
> @suhail-codenia I would recommend reading the docs, it has all the details. I would start here https://livewire.laravel.com/docs/quickstart

thanks buddy .
